packages/editor-ui/src/components/RunData.vue

```diff
@@ -199,8 +199,14 @@
 				</n8n-text>
 			</div>
 
-			<div v-else-if="hasNodeRun && hasRunError" :class="$style.errorDisplay">
-				<NodeErrorView :error="workflowRunData[node.name][runIndex].error" />
+			<div v-else-if="hasNodeRun && hasRunError" :class="$style.center">
+				<n8n-text v-if="isPaneTypeInput" :class="$style.center" size="xlarge" tag="h3" bold>
```

Comment:
can use `n8n-heading` component... also size should match other views.. it's too big now 
like here https://github.com/n8n-io/n8n/blob/31b0798b0be7156fe8aa4d62ddc5fcbeda18df3b/packages/editor-ui/src/components/RunData.vue#L223

Reply:
kept `n8n-text` but set the correct size and removed bold so now it looks like the message on the right pane